src/ppx_deriving_yojson.ml

```diff
@@ -883,6 +886,17 @@ let on_str_decls f ~options ~path type_decls =
 let on_sig_decls f ~options ~path type_decls =
   List.concat (List.map (f ~options ~path) type_decls)
 
+(* outside of register-create to use our sanitize, not opened one from Ppx_deriving *)
```

Comment:
I don't understand this comment, what does it mean?

Reply:
Previously `core_type` was defined inline:
https://github.com/ocaml-ppx/ppx_deriving_yojson/blob/632b49c9d62d439d560f18edbd1f004b08bf1ab4/src/ppx_deriving_yojson.ml#L895-L899
Since I renamed `wrap_runtime` → `sanitize`, then there would be shadowing of the correct `sanitize` to be called if it were still inline there. Writing `sanitize` in there would incorrectly refer to `Ppx_deriving.sanitize` (since it's locally opened) instead of the one in this module.

Reply:
Thanks for the explanation, I get it now. I would phrase it as a comment on the `sanitize` call:

> (* Note: we are careful to call our sanitize function here, not Ppx_deriving.sanitize. *)

Reply:
That sound better indeed, done!